packages/data-provider/src/services/submittedDataService.ts

```diff
@@ -85,12 +145,60 @@ const service = (dependencies: BaseDependencies) => {
 			);
 
 			return {
-				data: parseSubmittedData(recordsPaginated),
+				data: recordsPaginated,
 				metadata: {
 					totalRecords,
 				},
 			};
 		},
+		deleteSubmittedDataBySystemId: async (
```

Comment:
for consistency with the other files
```suggestion

		deleteSubmittedDataBySystemId: async (
```